recipes/jsoncpp/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+#
+source activate "${CONDA_DEFAULT_ENV}"
+
+export CFLAGS="-fPIC ${CFLAGS}"
+export CXXFLAGS="-fPIC ${CXXFLAGS}"
+
+if [ "$(uname)" == "Darwin" ]
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+else
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+mkdir build
+cd build
+cmake ..\
+        -DCMAKE_C_COMPILER="${CC}" \
+        -DCMAKE_CXX_COMPILER="${CXX}" \
+        -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+        -DCMAKE_INSTALL_LIBDIR="${PREFIX}/lib" \
```

Comment:
Without specifying the `CMAKE_INSTALL_LIBDIR` it will try to tack on the architecture (64-bit) in some cases (e.g. Linux pkg-config files), but not all cases. Setting this explicitly avoids that problem.